monkey/monkey_island/cc/server_utils/encryptor.py

```diff
@@ -51,7 +51,17 @@ def dec(self, enc_message):
         enc_message = base64.b64decode(enc_message)
         cipher_iv = enc_message[0 : AES.block_size]
         cipher = AES.new(self._cipher_key, AES.MODE_CBC, cipher_iv)
-        return self._unpad(cipher.decrypt(enc_message[AES.block_size :]).decode())
+        try:
+            dec_message = self._unpad(cipher.decrypt(enc_message[AES.block_size :]).decode())
+            return dec_message
+        except UnicodeDecodeError:
+            print("monkey-island secret key does not match MongoDB encrypted key")
+            print("Possible solutions:")
+            print("Run'docker container start -a monkey-island'")
+            print("Kill and restart the MongoDB container.")
```

Comment:
Instead of printing to stdout, we should use the logger to log an error.

These instructions are too specific. The user may have deployed monkey on Windows or via an AppImage, not Docker. We can just log a message that explains what the error is, but not how to fix it. The instructions on fixing it can go somewhere in our documentation.